### PR TITLE
RELEASE.md: schedule 3.[1,2,3,4] releases

### PR DESCRIPTION
**Pull request by jan--f**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -5,61 +5,12 @@ This page describes the release process and the currently planned schedule for u
...
-| v2.50          | 2024-01-16                                 | Augustin Husson (GitHub: @nexucis)          |
-| v2.51          | 2024-03-07                                 | Bryan Boreham (GitHub: @bboreham)           |
-| v2.52          | 2024-04-22                                 | Arthur Silva Sens (GitHub: @ArthurSens)     |
-| v2.53 LTS      | 2024-06-03                                 | George Krajcsovits (GitHub: @krajorama)     |
-| v2.54          | 2024-07-17                                 | Bryan Boreham (GitHub: @bboreham)           |
-| v2.55          | 2024-09-17                                 | Bryan Boreham (GitHub: @bboreham)           |
+| release series | date of first pre-release (year-month-day) | release shepherd               |
+|----------------|--------------------------------------------|--------------------------------|
+| v3.0           | 2024-11-14                                 | Jan Fajerski (GitHub: @jan--f) |
+| v3.1           | 2024-12-17                                 | **volunteer welcome**          |
```
Comment (beorn7) [inline RELEASE.md:13]: How about this (based on various chats today):

```suggestion
| v3.1           | 2024-12-17                                 | Bryan Boreham (GitHub: @bboreham) |
| v3.2           | 2024-12-17                                 | Jan Fajerski (GitHub: @jan--f)   |
| v3.3           | 2024-12-17                                 | **volunteer welcome**          |
```

**Review by beorn7 — APPROVED** [review]
@bboreham should also approve, of course.

File: RELEASE.md [hunk 2]
```diff
@@ -5,61 +5,14 @@ This page describes the release process and the currently planned schedule for u
...
-| v2.51          | 2024-03-07                                 | Bryan Boreham (GitHub: @bboreham)           |
-| v2.52          | 2024-04-22                                 | Arthur Silva Sens (GitHub: @ArthurSens)     |
-| v2.53 LTS      | 2024-06-03                                 | George Krajcsovits (GitHub: @krajorama)     |
-| v2.54          | 2024-07-17                                 | Bryan Boreham (GitHub: @bboreham)           |
-| v2.55          | 2024-09-17                                 | Bryan Boreham (GitHub: @bboreham)           |
+| release series | date of first pre-release (year-month-day) | release shepherd                  |
+|----------------|--------------------------------------------|-----------------------------------|
+| v3.0           | 2024-11-14                                 | Jan Fajerski (GitHub: @jan--f)    |
+| v3.1           | 2024-12-17                                 | Bryan Boreham (GitHub: @bboreham) |
+| v3.2           | 2025-02-28                                 | Jan Fajerski (GitHub: @jan--f)    |
```
Comment (beorn7) [inline RELEASE.md:14]:
```suggestion
| v3.2           | 2025-01-28                                 | Jan Fajerski (GitHub: @jan--f)    |
```

**Review by beorn7 — CHANGES_REQUESTED** [review]
Detected a typo, see comment.

**Review by beorn7 — APPROVED** [review]
Of course, everyone being volunteered here needs to approve, too.

**Review by machine424 — APPROVED** [review]

**Review by bboreham — APPROVED** [review]

**Pull request merged**